cmd/fossa/cmd/test/test.go

```diff
@@ -18,8 +18,20 @@ import (
 	"github.com/fossas/fossa-cli/config"
 )
 
+const defaultTestTemplate = `Test Failed. {{.Count}} {{if gt .Count 1 -}} issues {{- else -}} issue {{- end}} found:
+{{- range $type, $issues := .NormalizedByType}}
+========================================================================
```

Comment:
I'm not a big fan of noisy output—even if it's for humans. I'm not against _one_ such line to divide sections, but do we need two? Perhaps whitespace is just as effective but easier to read.

Reply:
I toyed around with a few options but I do still prefer the look of the double lines, especially as the number of issues and types increases. If you feel strongly about a change here feel free to leave an example.

Reply:
Resolved offline, message format has been improved as much as possible.